src/Clip.cpp

```diff
@@ -76,9 +76,9 @@ void Clip::init_settings()
 	wave_color = Color((unsigned char)0, (unsigned char)123, (unsigned char)255, (unsigned char)255);
 
 	// Init crop settings
-	crop_gravity = GRAVITY_CENTER;
-	crop_width = Keyframe(-1.0);
-	crop_height = Keyframe(-1.0);
+	crop_gravity = GRAVITY_TOP_LEFT;
+	crop_width = Keyframe(1.0);
+	crop_height = Keyframe(1.0);
```

Comment:
If these parameters are going to be exported to OpenShot, as I said I think `GRAVITY_CENTER` is how most people would assume cropping works (especially since they won't have the ability to modify `crop_gravity`), with the range for `crop_x` and `crop_y` being `-1.0` to `1.0`.

If that's _not_ going to be the default, then I think `crop_gravity` needs to be exported as well, so it can be set from the GUI. 

If everyone looking to remove black bars from a video (most common use case for cropping) will have to set `crop_x` and `crop_y` to move the origin point, instead of just reducing either `crop_width` or `crop_height` to reduce the video size in the appropriate dimension, that doesn't seem especially user-friendly. (Assuming I've understood properly how these parameters work?

Like, say you've got a `1920`x`1080` video file, and the contents are a `1920` x `800` film matted from 2.4:1 to 16:9 — _super_ common case. Trimming those black bars **should** be as simple as setting `crop_height` to `0.74` (`800 / 1080`) and done. Everything else can be left alone. That's how I think anyone would _expect_ it to work, no? 

If the origin is the upper-left corner, then you have to both set `crop_height` to `0.74`, _and_ set `crop_y` to `.13` (`((1080-800) / 2)/1080`) to move the origin point down to the upper-left corner of the video content.